agent/utils/mocks/nswrapper_mocks.go

```diff
@@ -0,0 +1,63 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
missing copyright header. i think you should have something similar to https://github.com/aws/amazon-ecs-agent/blob/master/agent/eni/netlinkwrapper/generate_mocks_linux.go under agent/utils/nswrapper so you can generate the mock with `make generate` (and the mock will have the header like other generated mock)